static/pattern-library/sass/main-ltr.scss

```diff
@@ -14,7 +14,7 @@
 // ------------------------------
 // #UTILITIES
 // ------------------------------
-@import 'utilities/variables';
+@import 'utilities/variables-ltr';
```

Comment:
Is there also a separate "-rtl" file here? I wasn't sure why this one got the suffix and the other reference didn't.